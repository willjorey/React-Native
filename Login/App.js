import React, { Component } from 'react';
import { Provider } from 'react-redux';

import store from './store'; //Import the store
// import Home from './components/home' //Import the component file
import LoginScreen from './screens/LoginScreen';
import TitleScreen from './screens/TitleScreen';


export default class App extends Component {
    render() {
        return (
            <Provider store={store}>
                {/* <LoginScreen/> */}
                <TitleScreen/>
            </Provider>
        );
    }
}

