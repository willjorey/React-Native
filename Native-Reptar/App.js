import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import FetchTournaments from './components/FetchTournaments';
import HomeScreen from './components/HomeScreen';
import TournamentScreen from './components/TournamentScreen';
import { createStackNavigator } from 'react-navigation';

const NavigationApp = createStackNavigator({
  Home: HomeScreen,
  Tournament: TournamentScreen,
  },
  {
    initialRouteName: 'Home',
  }
);

export default class App extends React.Component {
  render() {
    return (<NavigationApp/>);
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
});