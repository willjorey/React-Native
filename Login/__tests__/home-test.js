import React from 'react';
import Home from '../components/home';
import renderer from 'react-test-renderer';
import App from '../App';

test('App Renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('test', () => {
    
    expect(1).toBe(1);
})