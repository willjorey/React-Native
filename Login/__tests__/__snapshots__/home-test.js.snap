// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 30,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 50,
        "fontWeight": "bold",
      }
    }
  >
    Login
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Enter Username
  </Text>
  <View
    style={
      Object {
        "borderColor": "#d6d7da",
        "borderRadius": 4,
        "borderWidth": 3,
        "width": 200,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      underlineColorAndroid="transparent"
      value="Some Text"
    />
  </View>
  <View
    style={
      Object {
        "borderColor": "#d6d7da",
        "borderRadius": 4,
        "borderWidth": 3,
        "width": 200,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      underlineColorAndroid="transparent"
      value="Enter Password"
    />
  </View>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 30,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 50,
        "fontWeight": "bold",
      }
    }
  >
    Login
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Enter Username
  </Text>
  <View
    style={
      Object {
        "borderColor": "#d6d7da",
        "borderRadius": 4,
        "borderWidth": 3,
        "width": 200,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      underlineColorAndroid="transparent"
      value="Some Text"
    />
  </View>
  <View
    style={
      Object {
        "borderColor": "#d6d7da",
        "borderRadius": 4,
        "borderWidth": 3,
        "width": 200,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      underlineColorAndroid="transparent"
      value="Enter Password"
    />
  </View>
</View>
`;
